.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.buildTags": "codegen",
+}
```

Comment:
Do not commit this please. Maybe let's add `.vscode/settings.json` to `.gitignore` file ?

Reply:
I'd even recommend adding `.vscode` to a global gitignore on your system. Otherwise every repo needs to add every IDE's config paths (`.vscode`, `.idea`, etc.) individually.

Reply:
Done, thanks!